src/fragments/lib/auth/flutter/getting_started/60_developerPreview.mdx

```diff
@@ -0,0 +1,9 @@
+<Callout warning>
+
+Web and desktop support for the Amplify Flutter Auth category is now in developer preview!  See the documentation for [project setup](/lib/project-setup/create-application) and the [Auth category](/lib/auth/getting-started) for more information about web and desktop support.
+
+We will be adding support for additional categories on a rolling basis.
+
+The developer preview release is not intended for production usage. Cached credentials for users who have already logged in on existing applications will not be used by the developer preview.
```

Comment:
I don't think this is true anymore since we'll ship Jordan's migration logic.

```suggestion
The developer preview release is not intended for production usage.
```